app/cells/decidim/homepage_proposals/content_blocks/proposals_slider_cell.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+module Decidim
+  module HomepageProposals
+    module ContentBlocks
+      class ProposalsSliderCell < Decidim::ViewModel
+        attr_accessor :glanced_proposals
+
+        include Cell::ViewModel::Partial
+        include Core::Engine.routes.url_helpers
+        include Decidim::CardHelper
+        include Decidim::IconHelper
+        include ActionView::Helpers::FormOptionsHelper
+        include Decidim::FiltersHelper
+        include Decidim::FilterResource
+        include Decidim::LayoutHelper
+
+        private
+
+        def content_block_settings
+          @content_block_settings ||= Decidim::ContentBlock.find_by(
+            manifest_name: "proposals_slider",
+            organization: current_organization
+          ).settings
+        end
+
+        def options_for_default_component
+          components = Decidim::Component.where(id: content_block_settings.linked_components_id.reject(&:blank?).map(&:to_i))
```

Comment:
```suggestion
          components = Decidim::Component.where(id: content_block_settings.linked_components_id.compact.map(&:to_i))
```

`compact` is equivalent to `reject(&:blank?)`.